client/src/css/app.sass

```diff
@@ -72,6 +72,13 @@ a[target="_blank"]::after
         .q-field__control
             height: auto
 
+th.text-right, th.text-left, th.text-center, .q-table tbody td, .q-table__bottom-item,
```

Comment:
Style Criteria section on the Publication configuration page.
![Screenshot showing 10px and 14px font sizes in the Style Criteria section](https://github.com/MESH-Research/Pilcrow/assets/2668987/a97d92a4-9fc6-49b2-b868-fcc4f7dd6627)

Reply:
The paragraph is now 16px but I'm still seeing the tooltip presented as 10px.

Reply:
The tooltip should be 16px now